packages/gatsby-theme-ui-blog/package.json

```diff
@@ -11,20 +11,23 @@
     "react-dom": "^16.8.0"
   },
   "devDependencies": {
+    "@types/react-helmet": "^5.0.15",
     "gatsby": "^2.13.73",
+    "gatsby-plugin-typescript": "^2.1.27",
     "react": "^16.9.0",
     "react-dom": "^16.9.0"
   },
   "dependencies": {
     "@emotion/core": "^10.0.0",
     "@mdx-js/react": "^1.0.0",
+    "@theme-ui/core": "^0.3.1",
+    "@theme-ui/mdx": "^0.3.0",
     "@theme-ui/preset-base": "^0.3.0",
     "gatsby-plugin-react-helmet": "^3.1.3",
     "gatsby-plugin-theme-ui": "^0.3.0",
     "gatsby-theme-blog-core": "^1.0.2",
     "gatsby-theme-ui-layout": "^0.3.0",
-    "react-helmet": "^5.2.1",
-    "theme-ui": "^0.3.1"
```

Comment:
We should still have `theme-ui` in dependencies.

It's imported in https://github.com/system-ui/theme-ui/blob/7ac73eb283cd0de484363544ab2792e37a9f8951/packages/gatsby-theme-ui-blog/src/gatsby-theme-blog-core/components/post.ts#L1.

And then we don't need core and mdx. 

Dependencies could look like this
```
  "dependencies": {
    "@emotion/core": "^10.0.0",
    "@mdx-js/react": "^1.0.0",
    "@theme-ui/preset-base": "^0.4.0-alpha.0",
    "gatsby-plugin-react-helmet": "^3.1.3",
    "gatsby-plugin-theme-ui": "^0.4.0-alpha.1",
    "gatsby-theme-blog-core": "^1.0.2",
    "gatsby-theme-ui-layout": "^0.4.0-alpha.0",
    "react-helmet": "^5.2.1",
    "theme-ui": "^0.4.0-alpha.1"
  },
```